provision_gke_cluster.sh

```diff
@@ -1,4 +1,4 @@
 # This script provisions a GKE cluster
 #! /bin/bash -e
 
-gcloud container clusters create $1 --num-nodes=1 --machine-type=n1-standard-1
+gcloud container clusters create $1 --num-nodes=1 --machine-type=n1-standard-1 --zone=$2
```

Comment:
we need to paas the cluster region also while creating a cluster else it will throw an error